package.json

```diff
@@ -3,14 +3,20 @@
   "version": "1.0.0",
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
+  "engines": {
+    "node": "8.11.3",
+    "npm": "5.6.0"
+  },
   "scripts": {
-    "start": "babel-node index.js --exec babel-node",
+    "start": "babel-node index.js",
     "dev": "NODE_ENV=development nodemon ./index.js --exec babel-node",
-    "lint": "eslint .",
     "pretest": "npm run posttest && sequelize --env test db:migrate && sequelize --env test db:seed:all",
     "test": "cross-env NODE_ENV=test nyc mocha --compilers js:babel-core/register ./test/**/*.js",
     "posttest": "sequelize --env test db:migrate:undo:all",
-    "coverage": "npm run pretest && cross-env NODE_ENV=test babel-node ./node_modules/babel-istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha && npm run posttest"
```

Comment:
reverse your coverage to this script to enable your travis build to pass.

Reply:
ok